frontend/app/views/event/page.scala.html

```diff
@@ -49,7 +49,12 @@ <h1 class="event-masthead__name qa-event-detail-name">@event.name.text</h1>
                     }
                 </div>
             }
-            <img class="event-header__logo" src="@Asset.at("images/providers/" + event.providerOpt.getOrElse(event.metadata.identifier) + ".svg")">
+            @if(event.providerOpt.getOrElse(event.metadata.identifier) == "5x15") {
```

Comment:
Life's too short. Gonna merge it, and never look back. 

![giphy](https://cloud.githubusercontent.com/assets/394376/7474301/55258016-f338-11e4-98d3-550ab21f8c9a.gif)